internal/shared/jwt.go

```diff
@@ -72,11 +72,6 @@ func (l LpaStoreClaims) Validate() error {
 		switch iss {
 		case mrlpa:
 			return errors.New("Subject is not a valid URN")
-		case sirius:
-			emailRegex := regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
-			if !emailRegex.MatchString(sub) {
-				return errors.New("Subject is not a valid email or URN")
-			}
 		}
 	}
```

Comment:
The Sirius subject should still be valid URN, so I think this can be simplified to:
```go
	if !isUrn {
		return errors.New("Subject is not a valid URN")
	}
```